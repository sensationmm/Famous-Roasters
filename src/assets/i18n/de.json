{
  "auth": {
    "cognito": {
      "errors": {
        "confirmUserInvalidParameter": {
          "ctaText": "Anmelden",
          "prompt": "E-mail-Adresse ungültig. Versuch eine andere E-mail-Adresse oder"
        },
        "confirmUserNotFound": {
          "ctaText": "Registrieren",
          "prompt": "E-mail-Adresse ungültig. Versuch eine andere E-mail-Adresse oder"
        },
        "forgotPasswordGeneric": {
          "ctaText": "Registrieren",
          "prompt": "E-mail-Adresse ungültig. Versuch eine andere E-mail-Adresse oder"
        },
        "generic": "Anmeldungsfehler",
        "incorrectUserNameOrPassword": "Ungültige Daten",
        "invalidCode": "Verifizierungscode ungültig oder abgelaufen. Bitte versuch mal einen neuen Code anzufragen.",
        "loginGeneric": {
          "ctaText": "Registrieren",
          "prompt": "E-mail-Adresse oder Passwort ungültig, bitte prüfen oder"
        },
        "registerGeneric": {
          "ctaText": "Anmelden",
          "prompt": "E-mail-Adresse oder Passwort ungültig, bitte prüfen oder"
        }
      }
    },
    "signIn": {
      "email": {
        "error": {
          "invalidFormat": "Falsche Formatierung",
          "required": "Trage bitte deine E-mail-Adresse ein"
        },
        "label": "Deine E-mail-Adresse",
        "placeholder": "Deine E-mail-Adresse"
      },
      "forgotPassword": {
        "action": "Passwort vergessen?"
      },
      "password": {
        "error": {
          "required": "Trage bitte dein Passwort ein"
        },
        "label": "Dein Passwort",
        "placeholder": "Dein Passwort"
      },
      "signUp": {
        "action": "Registrieren",
        "prompt": "Ich bin neu hier"
      },
      "title": "Anmelden"
    },
    "signUp": {
      "confirmSignUp": {
        "action": "E-mail-Adresse bestätigen"
      },
      "confirmTos": {
        "error": {
          "required": "You need to tick the check box below to agree with the Terms and Conditions and the Privacy Policy before we can proceed"
        },
        "label": "By signing up I agree with the Terms and conditions and Privacy Policy."
      },
      "cta": "Registrieren",
      "email": {
        "error": {
          "invalidFormat": "Invalid email format",
          "required": "Please input your email"
        },
        "label": "Deine E-mail-Adresse",
        "placeholder": "Deine E-mail-Adresse"
      },
      "password": {
        "error": {
          "maxLength": "Your password must not be longer than 98 chars",
          "minLength": "Your password must be at least 8 characters long",
          "mustContainLowercase": "Your password must contain at least one lowercase letter",
          "mustContainNumber": "Your password must contain at least one number",
          "mustContainSpecial": "Your password must contain at least one special character: ^$*.[]{}()?\"!@#%&/\\,><':;|_~`",
          "mustContainUppercase": "Your password must contain at least one uppercase letter",
          "noSpaces": "Your password cannot contain spaces",
          "required": "Please input your password"
        },
        "label": "Dein Passwort",
        "placeholder": "Dein Passwort"
      },
      "passwordRepeat": {
        "error": {
          "matchError": "The password does not match",
          "required": "Please repeat your password"
        },
        "label": "Passwort wiederholen",
        "placeholder": "Passwort wiederholen"
      },
      "signIn": {
        "action": "Anmelden",
        "prompt": "Du hast bereits ein Konto?"
      },
      "title": "Erstelle ein Konto",
      "withEmail": "Registrierung mit Email"
    },
    "welcome": "Willkommen bei 60beans"
  },
  "brand": {
    "copyright": "© {{year}} 60beans. Alle Rechte vorbehalten.",
    "name": "60beans",
    "withLove": "Hergestellt mit ❤️ in Berlin."
  },
  "cta": {
    "next": "Weiter",
    "prev": "Zurück"
  },
  "error": {
    "generic": {
      "actions": {
        "retry": "Erneut versuchen"
      },
      "text": "Ein Fehler ist aufgetreten"
    },
    "unavailable": {
      "text": "Bald verfügbar!"
    }
  },
  "footer": {
    "customerService": {
      "cta": "Kundensupport",
      "text": "Fragen? Kommentare? Bedenken? Oder möchtest du uns einfach ein paar schöne Worte senden?",
      "howTo": "Bitte klicke einfach unten rechts auf die Sprechblasen. Unser Team steht dir dort gerne zur Verfügung.",
      "title": "Kundensupport"
    },
    "newsletter": {
      "cta": "Anmelden",
      "disclaimer": "Hinweis: Sie können sich jederzeit abmelden. Siehe unsere Datenschutzrichtlinie und Bedingungen.",
      "inputLabel": "Deine E-Mail",
      "inputPlaceholder": "max.mustermann@email.com",
      "text": "Melde dich für den Newsletter an und erhalte, einmalige Angebote, Neuigkeiten, Tipps und Insights zum Thema Specialty Coffee und neuen Röstern. ",
      "title": "Newsletter"
    },
    "slogan": {
      "text": "Guten Kaffee besser finden."
    }
  },
  "guides": {
    "acidity":{
      "items": {
        "0": {
          "text": "Du magst Kaffees mit leichter Säure.",
          "title": "Birnen"
        },
        "1": {
          "text": "Du magst Kaffees mit erkennbarer Säure.",
          "title": "Himbeeren"
        },
        "2": {
          "text": "Du magst Kaffees mit deutlicher Säure.",
          "title": "Orangen"
        }
      },
      "overline": "Erklärung",
      "text": "Unsere Kaffeeexperten nutzen den Vergleich mit Obst und Früchten, um herauszufinden, wie sauer dein Kaffee sein darf.",
      "title": "Warum wir dich das fragen?",
      "trigger": "Warum wir dich das fragen?"
    },
    "adventurous":{
      "items": {
        "0": {
          "text": "Du magst Kaffees mit einem leichten Körper und einem flüssigen Mundgefühl.",
          "title": "Ich bleibe bei dem, was ich kenne"
        },
        "1": {
          "text": "Du bist offen dafür Kaffeesorten zu probieren von denen du vorher noch nicht gehört hast.",
          "title": "Ich probiere etwas Neues aus"
        }
      },
      "overline": "Erklärung",
      "text": "Unsere Kaffeeexperten nutzen diese Frage, um herauszufinden, wie offen für Neues du bist.",
      "title": "Warum wir dich das fragen?",
      "trigger": "Warum wir dich das fragen?"
    },
    "bitterness":{
      "items": {
        "0": {
          "text": "Du magst Kaffee mit einer leichten Bitternote.",
          "title": "Milchschokolade"
        },
        "1": {
          "text": "Du magst Kaffees mit einer ausgeglichenen Bitternote.",
          "title": "Schokolade mit hohem Kakaoanteil (<70%)"
        },
        "2": {
          "text": "Du magst Kaffees mit einer starken Bitternote.",
          "title": "Dunkle Schokolade"
        }
      },
      "overline": "Erklärung",
      "text": "Unsere Kaffeeexperten nutzen den Vergleich mit Schokolade, um herauszufinden, wie bitter dein Kaffee sein darf.",
      "title": "Warum wir dich das fragen?",
      "trigger": "Warum wir dich das fragen?"
    },
    "body":{
      "items": {
        "0": {
          "text": "Du magst Kaffees mit einem leichten Körper und einem flüssigen Mundgefühl.",
          "title": "Leichter Körper und flüssiges Mundgefühl - Americano"
        },
        "1": {
          "text": "Du magst Kaffees mit einem leichten Körper und einem milchigen Mundgefühl.",
          "title": "Ausgewogener Körper mit einem milchigen Mundgefühl - Milchkaffee"
        },
        "2": {
          "text": "Du magst Kaffees mit einem vollen Körper und einem cremigen Mundgefühl.",
          "title": "Ausgewogener Körper mit einem cremigen Mundgefühl - Macchiato "
        }
      },
      "overline": "Erklärung",
      "text": "Unsere Kaffeeexperten nutzen deine Zubereitungspräferenz, um herauszufinden, wie der Körper bzw. das Mundgefühl deines Kaffees sein soll.",
      "title": "Warum wir dich das fragen?",
      "trigger": "Warum wir dich das fragen?"
    },
    "grindType":{
      "overline": "Erklärung",
      "text": "Da wir dir am Ende nicht nur neue, sondern auch möglichst passende Kaffeesorten vorschlagen möchten, müssen wir wissen, wie du deinen Kaffee am liebsten zubereitest.",
      "textBold": "Das heißt: wir empfehlen dir keinen Filterkaffee, wenn du einen Porta-Filter benutzt.",
      "title": "Warum wir dich das fragen?",
      "trigger": "Warum wir dich das fragen?"
    },
    "sweetness":{
      "items": {
        "0": {
          "text": "Du magst Kaffees mit einer sehr geringen Süße.",
          "title": "Käse"
        },
        "1": {
          "text": "Du magst Kaffees mit einer erkennbaren Süße.",
          "title": "Panna Cotta"
        },
        "2": {
          "text": "Du magst Kaffees mit einer sehr starken Süße.",
          "title": "Baklava"
        }
      },
      "overline": "Erklärung",
      "text": "Unsere Kaffeeexperten nutzen den Vergleich mit Süßspeisen, um herauszufinden, wie süß dein Kaffee sein darf.",
      "title": "Warum wir dich das fragen?",
      "trigger": "Warum wir dich das fragen?"
    },
    "tasteResults": {
      "items": {
        "acidity": {
          "title": "Säure",
          "text": {
            "hi": "Du magst Kaffees mit deutlicher Säure.",
            "lo": "Du magst Kaffees mit leichter Säure.",
            "mid": "Du magst Kaffees mit erkennbarer Säure."
          }
        },
        "bitterness": {
          "title": "Stärke",
          "text": {
            "hi": "Du magst Kaffees mit einer starken Bitternote",
            "lo": "Du magst Kaffee mit einer leichten Bitternote.",
            "mid": "Du magst Kaffees mit einer ausgeglichenen Bitternote."
          }
        },
        "body": {
          "title": "Körper",
          "text": {
            "hi": "Du magst Kaffees mit einem vollen Körper und einem cremigen Mundgefühl.",
            "lo": "Du magst Kaffees mit einem leichten Körper und einem flüssigen Mundgefühl.",
            "mid": "Du magst Kaffees mit einem leichten Körper und einem milchigen Mundgefühl."
          }
        },
        "sweetness": {
          "title": "Süße",
          "text": {
            "hi": "Du magst Kaffees mit einer sehr starken Süße.",
            "lo": "Du magst Kaffees mit einer sehr geringen Süße.",
            "mid": "Du magst Kaffees mit einer erkennbaren Süße."
          }
        }
      },
      "overline": "Erklärung",
      "text": "Dein Kaffeetyp zeigt wie du deinen Kaffee magst. Diese Eigenschaften zeigen, wie dein optimaler Kaffeegenuss schmecken soll.",
      "title": "Was bedeutet mein Kaffeetyp?",
      "trigger": "Was bedeutet mein Kaffeetyp?"
    }
  },
  "pages": {
    "about": {
      "navigation": "Über uns",
      "title": "Über uns"
    },
    "account": {
      "navigation": "Konto",
      "title": "Konto"
    },
    "cart": {
      "ctaCheckout": "Zur Kasse",
      "ctaContinue": "Weiter einkaufen",
      "displayTitle": "Dein Warenkorb",
      "navigation": "Warenkorb",
      "noItems": "Dein Warenkorb ist leer",
      "notification": {
        "add": {
          "body": "Dein Kaffee wartet schon auf dich",
          "heading": "Zum Warenkorb hinzugefügt"
        }
      },
      "subtotal": "Zwischensumme",
      "title": "Warenkorb"
    },
    "catalogue": {
      "filters": {
        "beanType": {
          "label": "Bohnensorte"
        },
        "coffeeType": {
          "label": "Alle Zubereitungsmethoden",
          "values": {
            "none": "Alle Zubereitungsmethoden"
          }
        },
        "common": {
          "filtersMenu": {
            "close": "Schließen",
            "filter": "Filter",
            "filterOptions": "Filteroptionen",
            "open": "Öffnen",
            "removeFilter": "Filter entfernen",
            "showResults": "Ergebnisse anzeigen"
          },
          "filterMobile": {
            "apply": "Übernehmen"
          }
        },
        "decaf": "Nur Entkoffeiniert",
        "origin": {
          "label": "Herkunft",
          "values": {
            "AC":"Ascension",
            "AD":"Andorra",
            "AE":"Vereinigte Arabische Emirate",
            "AF":"Afghanistan",
            "AG":"Antigua und Barbuda",
            "AI":"Anguilla",
            "AL":"Albanien",
            "AM":"Armenien",
            "AO":"Angola",
            "AQ":"Antarktis",
            "AR":"Argentinien",
            "AS":"Amerikanisch-Samoa",
            "AT":"Österreich",
            "AU":"Australien",
            "AW":"Aruba",
            "AX":"Åland",
            "AZ":"Aserbaidschan",
            "BA":"Bosnien und Herzegowina",
            "BB":"Barbados",
            "BD":"Bangladesch",
            "BE":"Belgien",
            "BF":"Burkina Faso",
            "BG":"Bulgarien",
            "BH":"Bahrain",
            "BI":"Burundi",
            "BJ":"Benin",
            "BL":"Saint-Barthélemy",
            "BM":"Bermuda",
            "BN":"Brunei",
            "BO":"Bolivien",
            "BQ":"Bonaire, Saba, Sint Eustatius",
            "BR":"Brasilien",
            "BS":"Bahamas",
            "BT":"Bhutan",
            "BU":"Myanmar",
            "BV":"Bouvetinsel",
            "BW":"Botswana",
            "BY":"Belarus",
            "BZ":"Belize",
            "CA":"Kanada",
            "CC":"Kokosinseln",
            "CD":"Kongo",
            "CF":"Zentralafrikanische Republik",
            "CG":"Kongo",
            "CH":"Schweiz",
            "CI":"Elfenbeinküste",
            "CK":"Cookinseln",
            "CL":"Chile",
            "CM":"Kamerun",
            "CN":"China",
            "CO":"Kolumbien",
            "CP":"Clipperton",
            "CR":"Costa Rica",
            "CU":"Kuba",
            "CV":"KapVerde",
            "CW":"Curaçao",
            "CX":"Weihnachtsinsel",
            "CY":"Zypern",
            "CZ":"Tschechien",
            "DE":"Deutschland",
            "DG":"Diego Garcia",
            "DJ":"Dschibuti",
            "DK":"Dänemark",
            "DM":"Dominica",
            "DO":"Dominikanische Republik",
            "DZ":"Algerien",
            "EA":"Ceuta, Melilla",
            "EC":"Ecuador",
            "EE":"Estland",
            "EG":"Ägypten",
            "EH":"Westsahara",
            "ER":"Eritrea",
            "ES":"Spanien",
            "ET":"Äthiopien",
            "FI":"Finnland",
            "FJ":"Fidschi",
            "FK":"Falklandinseln",
            "FM":"Mikronesien",
            "FO":"Färöer",
            "FR":"Frankreich",
            "GA":"Gabun",
            "GB":"Vereinigtes Königreich",
            "GD":"Grenada",
            "GE":"Georgien",
            "GF":"Französisch-Guayana",
            "GG":"Guernsey ",
            "GH":"Ghana",
            "GI":"Gibraltar",
            "GL":"Grönland",
            "GM":"Gambia",
            "GN":"Guinea",
            "GP":"Guadeloupe",
            "GQ":"Äquatorialguinea",
            "GR":"Griechenland",
            "GS":"Südgeorgien und die Südlichen Sandwichinseln",
            "GT":"Guatemala",
            "GU":"Guam",
            "GW":"Guinea-Bissau",
            "GY":"Guyana",
            "HK":"Hongkong",
            "HM":"Heard und Mc Donald inseln",
            "HN":"Honduras",
            "HR":"Kroatien",
            "HT":"Haiti",
            "HU":"Ungarn",
            "IC":"KanarischeInseln",
            "ID":"Indonesien",
            "IE":"Irland",
            "IL":"Israel",
            "IM":"Isle of Man",
            "IN": "Indien",
            "IO":"Britisches Territorium im Indischen Ozean",
            "IQ":"Irak",
            "IR":"Iran",
            "IS":"Island",
            "IT":"Italien",
            "JE":"Jersey",
            "JM":"Jamaika",
            "JO":"Jordanien",
            "JP":"Japan",
            "KE":"Kenia",
            "KG":"Kirgisistan",
            "KH":"Kambodscha",
            "KI":"Kiribati",
            "KM":"Komoren",
            "KN":"St. Kitts und Nevis",
            "KP":"Nordkorea",
            "KR":"Südkorea",
            "KW":"Kuwait",
            "KY":"CaymanIslands",
            "KZ":"Kasachstan",
            "LA":"Laos",
            "LB":"Libanon",
            "LC":"St. Lucia",
            "LI":"Liechtenstein",
            "LK":"SriLanka",
            "LR":"Liberia",
            "LS":"Lesotho",
            "LT":"Litauen",
            "LU":"Luxemburg",
            "LV":"Lettland",
            "LY":"Libyen",
            "MA":"Marokko",
            "MC":"Monaco",
            "MD":"Moldau",
            "ME":"Montenegro",
            "MF":"Saint-Martin",
            "MG":"Madagaskar",
            "MH":"Marshallinseln",
            "MK":"Nordmazedonien",
            "ML":"Mali",
            "MM":"Myanmar",
            "MN":"Mongolei",
            "MO":"Macau",
            "MP":"NördlicheMarianen",
            "MQ":"Martinique",
            "MR":"Mauretanien",
            "MS":"Montserrat",
            "MT":"Malta",
            "MU":"Mauritius",
            "MV":"Malediven",
            "MW":"Malawi",
            "MX":"Mexiko",
            "MY":"Malaysia",
            "MZ":"Mosambik",
            "NA":"Namibia",
            "NC":"Neukaledonien",
            "NE":"Niger",
            "NF":"Norfolkinsel",
            "NG":"Nigeria",
            "NI":"Nicaragua",
            "NL":"Niederlande",
            "NO":"Norwegen",
            "NP":"Nepal",
            "NR":"Nauru",
            "NT":"NeutraleZone",
            "NU":"Niue",
            "NZ":"Neuseeland",
            "OM":"Oman",
            "PA":"Panama",
            "PE":"Peru",
            "PF":"Französisch-Polynesien",
            "PG":"Papua-Neuguinea",
            "PH":"Philippinen",
            "PK":"Pakistan",
            "PL":"Polen",
            "PM":"Saint-Pierre und Miquelon",
            "PN":"Pitcairninseln",
            "PR":"PuertoRico",
            "PS":"Palästina",
            "PT":"Portugal",
            "PW":"Palau",
            "PY":"Paraguay",
            "QA":"Katar",
            "RE":"Réunion",
            "RO":"Rumänien",
            "RS":"Serbien",
            "RU":"Russland",
            "RW":"Ruanda",
            "SA":"Saudi-Arabien",
            "SB":"Salomonen",
            "SC":"Seychellen",
            "SD":"Sudan",
            "SE":"Schweden",
            "SG":"Singapur",
            "SH":"St. Helena, Ascension und Tristanda Cunha",
            "SI":"Slowenien",
            "SJ":"Spitzbergen und Jan Mayen",
            "SK":"Slowakei",
            "SL":"SierraLeone",
            "SM":"San Marino",
            "SN":"Senegal",
            "SO":"Somalia",
            "SR":"Suriname",
            "SS":"Südsudan",
            "ST":"São Tomé und Príncipe",
            "SV":"El Salvador",
            "SX":"SintMaarten",
            "SY":"Syrien",
            "SZ":"Eswatini",
            "TA":"Tristanda Cunha",
            "TC":"Turks-undCaicosinseln",
            "TD":"Tschad",
            "TF":"Französische Süd- und Antarktisgebiete",
            "TG":"Togo",
            "TH":"Thailand",
            "TJ":"Tadschikistan",
            "TK":"Tokelau",
            "TM":"Turkmenistan",
            "TN":"Tunesien",
            "TO":"Tonga",
            "TR":"Türkei",
            "TT":"Trinidad und Tobago",
            "TV":"Tuvalu",
            "TW":"Taiwan",
            "TZ":"Tansania",
            "UA":"Ukraine",
            "UG":"Uganda",
            "UM":"United States Minor Outlying Islands",
            "US":"Vereinigte Staaten",
            "UY":"Uruguay",
            "UZ":"Usbekistan",
            "VA":"Vatikanstadt",
            "VC":"St. Vincent und die Grenadinen",
            "VE":"Venezuela",
            "VG":"Britische Jungferninseln",
            "VI":"Amerikanische Jungferninseln",
            "VN":"Vietnam",
            "VU":"Vanuatu",
            "WF":"Wallis und Futuna",
            "WS":"Samoa",
            "YE":"Jemen",
            "YT":"Mayotte",
            "ZA":"Südafrika",
            "ZM":"Sambia",
            "ZR":"Kongo",
            "ZW":"Simbabwe"
          }
        },
        "packageSize": {
          "label": "Packungsgröße"
        },
        "pricePerKg": {
          "label": "Preis je Kg"
        },
        "sort": {
          "label": "Sortieren nach...",
          "values": {
            "priceAsc": "Preis aufsteigend",
            "priceDesc": "Preis absteigend",
            "newDesc": "Neue zuerst",
            "none": "Standard"
          }
        },
        "vendor": {
          "label": "Röster"
        }
      },
      "navigation": "Shop",
      "pagination": {
        "next": "Weiter",
        "previous": "Zurück"
      },
      "tabs": {
        "discover": "Entdecken",
        "forYou": "Für Dich"
      },
      "tile": {
        "from": "ab"
      },
      "title": "Katalog"
    },
    "checkout": {
      "navigation": "Checkout",
      "title": "Checkout"
    },
    "contact": {
      "navigation": "Kontakt",
      "title": "Kontakt"
    },
    "error": {
      "navigation": "Fehler",
      "title": "Fehler"
    },
    "featuredProduct": {
      "cta": {
        "discoverMore": "Mehr Kaffeesorten entdecken",
        "goToProduct": "Zum Produkt"
      },
      "items": {
        "chocolate": {
          "description": "Dein Kaffee ist säurearm, hat einen nussig-süßen Geschmack und ist oft bittersüß mit einem schokoladigen Röstgeschmack."
        },
        "experimental": {
          "description": "Dein Kaffee ist komplex, er zeigt einen mittleren Körper mit zitronigen Säurenoten. Die Frische und Lebendigkeit machen jede Tasse einzigartig."
        },
        "floral": {
          "description": "Dein Kaffee ist süß, leicht und trägt einen subtilen Duft, ähnlich wie Blumen. Er bringt fruchtige Töne, helle Säure und einen mittleren bis schweren Körper zusammen."
        },
        "fruits": {
          "description": "Weder subtil oder zart, Dein Kaffee ist frisch und lebendig. Du magst das Mundgefühl mit Schärfe, einem kräftigen Körper und fruchtigen Geschmacksnoten."
        },
        "spicy": {
          "description": "Du liebst die rustikale Note und eine Vollkornbrotsüße. Er zeigt eine ausgewogene Säure mit einem kräftigen Körper und intensiven Aromen."
        }
      },
      "navigation": "Aroma",
      "recommendation": {
        "title": "Was wir dir empfehlen",
        "facts": "Warum du ihn lieben wirst"
      },
      "title": "Aroma",
      "yourCoffeeType": {
        "title": "Dein Kaffeetyp"
      }
    },
    "home": {
      "navigation": "Home",
      "title": "Home"
    },
    "imprint": {
      "navigation": "Impressum",
      "title": "Impressum"
    },
    "jobs": {
      "navigation": "Jobs",
      "title": "Jobs"
    },
    "ourRoasters": {
      "navigation": "Unsere Röster",
      "title": "Unsere Röster"
    },
    "privacy": {
      "navigation": "Datenschutz-Bestimmungen",
      "title": "Datenschutz-Bestimmungen"
    },
    "press": {
      "navigation": "Presse",
      "title": "Presse"
    },
    "product": {
      "navigation": "Produkt",
      "originProcessing": {
        "characteristics": {
          "altitude": "Anbauhöhe",
          "flavourNotes": "Geschmacksnoten",
          "origin": "Herkunftsland",
          "processing": "Verarbeitung",
          "producer": "Produzent",
          "variety": "Sorte"
        },
        "title": "Herkunft und Verarbeitung"
      },
      "title": "Produkt",
      "sections": {
        "findSimilar": {
          "title": "Ähnliche Produkte"
        },
        "getToKnow": {
          "title": "Alles zum Kaffee"
        },
        "learnToBrew": {
          "title": "Zubereitungstipps"
        },
        "meetTheRoaster": {
          "title": "Rösterei im Profil"
        },
        "tasteProfile": {
          "parameter": {
            "acidity": {
              "description": "Kaffeebohnen enthalten von Natur aus Säure. Unsere Kaffeeexperten vergleichen den Säuregehalt auch mit Obst und Früchten, um herauszufinden, wie sauer dein Kaffee sein darf.",
              "name": "Säure"
            },
            "bitterness": {
              "description": "Die Stärke der Röstung beeinflusst den Grad der Bitterstoffe im Kaffee. Eine helle Röstung entspricht der Bitternote einer hellen Milchschokolade. Eine intensive Röstung entspricht einer dunkleren Schokolade.",
              "name": "Stärke"
            },
            "body": {
              "description": "Unsere Kaffeeexperten nutzen den Begriff Körper um das Mundgefühl des Kaffees zu beschreiben. Unter einem vollen Körper kannst du dir einen vollmundigen Espresso vorstellen. Ein leichter Körper entspricht eher einem leichten Americano.",
              "name": "Körper"
            },
            "sweetness": {
              "description": "Die Süße des Kaffees wird maßgeblich durch die Kaffeebohne, die Anbauhöhe und die Art der Röstung beeinflußt.\n\n",
              "name": "Süße"
            }
          },
          "scale": {
            "1": "Leicht",
            "2": "Mittel",
            "3": "Hoch"
          }
        }
      },
      "transactional": {
        "cta": "In den Warenkorb",
        "options": {
          "grindType": {
            "label": "Mahlgrad",
            "moreInfo": "Mehr zu den Mahlgraden",
            "guide": {
              "title": "Mahlgraden Handbuch",
              "items": {
                "aeropress": {
                  "text": "Mittlere Körnung, etwas feiner als Sandkörner.",
                  "title": "Aeropress"
                },
                "chemex": {
                  "text": "Mittelgrob gemahlen. Ungefähr wie grober Sand.",
                  "title": "Chemex"
                },
                "espresso": {
                  "text": "Noch feiner gemahlen als Speisesalz.",
                  "title": "Espresso"
                },
                "french": {
                  "text": "Grob gemahlen, ähnlich wie Meersalz.",
                  "title": "French Press"
                },
                "moka": {
                  "text": "Mittelfeiner Mahlgrad für Mokkakannen.",
                  "title": "Moka Pot"
                },
                "v60": {
                  "text": "Mittelgrob gemahlen. Ungefähr wie grober Sand.",
                  "title": "V60 / Kalita"
                },
                "whole": {
                  "text": "Kann als Ganzes oder gemahlen verwendet werden. Je nach dem was du lieber magst.",
                  "title": "Ganze Bohne"
                }
              }
            }
          },
          "packageSize": {
            "label": "Größe/Menge"
          },
          "quantity": {
            "label": "Anzahl"
          }
        },
        "price": {
          "footNote": "Preise inkl. MwSt."
        },
        "qualityNote": "Der Röster röstet deinen Kaffee frisch und schickt dein Paket so früh wie möglich direkt zu Dir. Wir teilen Dir den geplanten Liefertermin im nächsten Schritt mit.",
        "shipping": {
          "label": "Lieferzeit",
          "value": "2-3 Tage"
        }
      }
    },
    "tasteFinder": {
      "navigation": "Tastefinder",
      "title": "Tastefinder",
      "steps": {
        "acidity": {
          "options": {
            "option1": {
              "selectedText": "Leichte Säure",
              "text": "Weniger saure Birnen"
            },
            "option2": {
              "selectedText": "Mittlere Säure",
              "text": "Ausgewogen saure Himbeeren"
            },
            "option3": {
              "selectedText": "Intensive Säure",
              "text": "Sehr saure Orangen"
            }
          },
          "text": "Sonntagmorgen. Die Vögel zwitschern und die Sonne scheint. Du bereitest dir einen Obstsalat zu, aber ein Obst fehlt noch. Welches Obst darf in deinem Salat nicht fehlen?"
        },
        "adventurous": {
          "options": {
            "option1": {
              "text": "Ich bleibe bei dem, was ich kenne"
            },
            "option2": {
              "text": "Ich probiere etwas Neues aus"
            }
          },
          "text": "Oh nein. Alles ausgetrunken. Du hast keinen Kaffee mehr. Was würdest du jetzt normalerweise tun?"
        },
        "bitterness": {
          "options": {
            "option1": {
              "selectedText": "Leichte Bitternote",
              "text": "Hellere und weniger bittere Milchschokolade"
            },
            "option2": {
              "selectedText": "Ausgeglichene Bitternote",
              "text": "Dunklere und ausgewogen bittere Schokolade"
            },
            "option3": {
              "selectedText": "Intensive Bitternote",
              "text": "Sehr dunkle Zartbitter-schokolade (70% +)"
            }
          },
          "text": "Stell dir vor, dass verschiedene Sorten Schokolade vor dir auf dem Tisch liegen. Für welche würdest du dich entscheiden?"
        },
        "body": {
          "options": {
            "option1": {
              "selectedText": "Leichter Körper",
              "text": "Americano - leichter Körper, flüssiges Mundgefühl"
            },
            "option2": {
              "selectedText": "Ausgewogener Körper",
              "text": "Milchkaffee - ausgewogener Körper, milchiges Mundgefühl "
            },
            "option3": {
              "selectedText": "Voller Körper",
              "text": "Macchiato - voller Körper, cremiges Mundgefühl"
            }
          },
          "text": "Keine Termine und viel Zeit für eine gute Tasse Kaffee. Wie würdest du deinen Kaffee bestellen?"
        },
        "grindType": {
          "options": {
            "option1": {
              "text": "French Press"
            },
            "option2": {
              "text": "Aeropress"
            },
            "option3": {
              "text": "V60/Kalita"
            },
            "option4": {
              "text": "Moka Pot"
            },
            "option5": {
              "text": "Chemex"
            },
            "option6": {
              "text": "Espresso Maschine"
            }
          },
          "text": "Jeder weiß: der Tag beginnt nach der ersten Tasse Kaffee. Auf welche Art brühst du deinen Kaffee?"
        },
        "processing": {
          "loading1": "Kurze Kaffeepause.",
          "loading2": "Wir stellen basierend auf deiner Auswahl dein Kaffeeprofil zusammen."
        },
        "sweetness": {
          "options": {
            "option1": {
              "selectedText": "Leichte Süße",
              "text": "Eine leicht süßes Stück Käse"
            },
            "option2": {
              "selectedText": "Mittlere Süße",
              "text": "Eine ausgewogen süße Panna Cotta"
            },
            "option3": {
              "selectedText": "Intensive Süße",
              "text": "Ein intensiv süßes Stück Baklava"
            }
          },
          "text": "Der Tag neigt sich dem Ende zu und du hast noch etwas Appetit. Für welchen Snack entscheidest du dich?"
        },
        "welcome": {
          "cta": "Tastefinder starten",
          "text": "Finde deinen persönlichen Kaffeetyp und erhalte Empfehlungen, die zu deinem Geschmack passen.",
          "title": "Finde jetzt deinen Specialty Coffee mit dem Tastefinder."
        },
        "yourName": {
          "input": {
            "label": "Gib deinen Namen ein",
            "placeholder": "Dein Name"
          },
          "title": "Wie sollen wir dich nennen?"
        }
      }
    }
  }
}