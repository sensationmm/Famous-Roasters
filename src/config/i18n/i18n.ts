import i18n from 'i18next'
import { initReactI18next } from 'react-i18next'

const resources = {
  de: {
    translation: require('src/assets/i18n/de.json'),
  },
}

i18n.use(initReactI18next).init({
  resources,
  fallbackLng: 'de',
  keySeparator: '.',
  interpolation: {
    escapeValue: false,
  },
  detection: {
    order: ['localStorage', 'navigator'],
    caches: ['localStorage'],
  },
})

export default i18n
