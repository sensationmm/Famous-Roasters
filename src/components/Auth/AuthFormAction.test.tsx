import { MockedProvider } from '@apollo/client/testing'
import { render, waitFor } from '@testing-library/react'
import React from 'react'
import { I18nextProvider } from 'react-i18next'
import i18n from 'src/config/i18n'

import { AuthFormAction } from '.'

describe('Auth form action custom renderer on auth component', () => {
  const snippet = () => (
    <I18nextProvider i18n={i18n}>
      <form name="testAuthFormAction">
        <AuthFormAction promptText="promptText" onClick={() => null} dataTestId="testId" ctaText="ctaText" />
      </form>
    </I18nextProvider>
  )

  it('Renders correctly', async () => {
    const { container } = render(
      <MockedProvider defaultOptions={{ watchQuery: { fetchPolicy: 'no-cache' } }}>{snippet()}</MockedProvider>,
    )
    await waitFor(() => new Promise((res) => setTimeout(res, 0)))
    expect(container).toMatchSnapshot()
  })
})
