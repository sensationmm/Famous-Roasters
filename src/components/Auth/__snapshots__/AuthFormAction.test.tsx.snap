// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Auth form action custom renderer on auth component Renders correctly 1`] = `
<div>
  <form
    name="testAuthFormAction"
  >
    <div
      class="text-xs leading-4 font-semibold"
    >
      promptText 
      <a
        data-testid="testId"
      >
        ctaText
      </a>
    </div>
  </form>
</div>
`;
